apps/antalmanac/tsconfig.json

```diff
@@ -22,7 +22,7 @@
             "$lib/*": ["src/lib/*"],
             "$providers/*": ["src/providers/*"],
             "$routes/*": ["src/routes/*"],
-            "$stores/*": ["src/stores/*"],
```

Comment:
Extra commas is invalid in JSON; TypeScript doesn't mind when parsing the file, but following the spec is good.